### ISSUE TITLE
Question about string: 'Average Spending Per Pupil'

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/least-squares-regression#67]
Submitted by Sandor.

@amanda-phet Can you comment on exactly what this string means? I dug around a bit and it seems that there are two metrics: "Expenditure per pupil in average daily attendance" and "Expenditure per pupil in fall enrollment" is it one of these, or another?

This is relevant for translations of these strings. 

I looked in the design doc and couldn't find the exact source. 

>  have some questions about the Least square regression sim I am translating.
> Look at this string in the sim:
> Average Spending Per Pupil
> Is it the money spent by a pupil or the money?
> Or is it the money spent on the pupil [by parents(?), school(?), the teacher(?)] ?

**Comment by amanda-phet** [comment]
"average spending per pupil" is the state expenditure per student in a given school year. So, the amount a state spends per enrolled student per year.

**Comment by oliver-phet** [comment]
Should this be clarified in the '?' popup that the sim provides? It seems pretty ambiguous to me and it is probably even more so to translators.

**Comment by amanda-phet** [comment]
Should we say "expenditure" instead of "spending"?

**Comment by oliver-phet** [comment]
What about "average state expenditure per pupil"?

**Comment by amanda-phet** [comment]
It is a long string for the y-axis, but should be fine. 

Should we also change the dialog to say: "average per-pupil expenditure is compared to average teacher salaries in fifty US states in 1986."?

**Comment by oliver-phet** [comment]
I think that could work!

**Comment by oliver-phet** [comment]
What about paralleling this dialog to the 'Life Expectancy vs TVs':
"Average per-pupil expenditure in 50 US states is compared to average teacher salaries in those states in 1986."?

**Comment by amanda-phet** [comment]
Sounds great. Thanks for your suggestions @oliver-phet .

**Comment by veillette** [comment]
The title and y-axis were changed. Assigning to @amanda-phet  for review.

**Comment by amanda-phet** [comment]
Looks good! 

@oliver-phet do you want to let Sandor know about the change once a maintenance release happens?

**Comment by oliver-phet** [comment]
Yes, I'll email him. Can you let me know when the maintenance release happens?

**Comment by amanda-phet** [comment]
Sure. I have no idea when that will happen, but I'll let you know when it does, and will keep this issue open to remind me.

**Comment by amanda-phet** [comment]
Un-assigning until this sim has a maintenance release.
